scripts/observability/sentry_release.sh

```
@@ -1,33 +1,87 @@
-project="${SENTRY_PROJECT_XAI_DISSECT:-}"
+project="${SENTRY_PROJECT_XAI_DISSECT:-${SENTRY_PROJECT:-}}"
 environment="${SENTRY_ENVIRONMENT:-local}"
-git_sha="${AGENTOS_GIT_SHA:-$(git rev-parse --short HEAD 2>/dev/null || printf 'unknown')}"
+# Match runtime: xai-dissect@<AGENTOS_GIT_SHA|unknown> (see observability::git_sha).
+git_sha="${AGENTOS_GIT_SHA:-unknown}"
+# Trim leading/trailing whitespace to match runtime git_sha_from_value (str::trim).
+git_sha="$(printf '%s' "${git_sha}" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
+if [[ -z "${git_sha}" ]]; then
+  git_sha="unknown"
+fi
+fi
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)